pages/installation/os/sles.rst

```diff
@@ -0,0 +1,122 @@
+
+    [elasticsearch-5.x]
+    name=Elasticsearch repository for 5.x packages
+    baseurl=https://artifacts.elastic.co/packages/5.x/yum
+    gpgcheck=1
+    gpgkey=https://artifacts.elastic.co/GPG-KEY-elasticsearch
+    enabled=1
+    autorefresh=1
+    type=rpm-md
+
+followed by the installation of the latest release with ``sudo zypper install elasticsearch``.
+
+Make sure to modify the `Elasticsearch configuration file <https://www.elastic.co/guide/en/elasticsearch/reference/5.6/settings.html#settings>`__  (``/etc/elasticsearch/elasticsearch.yml``) and set the cluster name to ``graylog`` additionally you need to uncomment (remove the # as first character) the line::
+
+    cluster.name: graylog
+
+After you have modified the configuration, you can start Elasticsearch::
+
+    $ sudo chkconfig elasticsearch on
+    $ sudo systemctl daemon-reload
+    $ sudo systemctl restart elasticsearch.service
+
+
+Graylog
+-------
+
+First install the Graylog GPG Key with ``rpm --import https://git.io/vdPub`` then add the repository file ``/etc/zypp/repos.d/graylog.repo`` with the following content::
+
+    [graylog]
+    name=graylog
+    baseurl=https://packages.graylog2.org/repo/el/stable/2.3/$basearch/
+    gpgcheck=1
+    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-graylog
+
+followed by the installation of the latest release with ``sudo zypper install graylog-server``.
+
+Make sure to follow the instructions in your ``/etc/graylog/server/server.conf`` and add ``password_secret`` and ``root_password_sha2``. These settings are mandatory and without them, Graylog will not start!
+
+You can use the following command to create your ``password_secret``::
+
+    cat /dev/urandom | base64 | cut -c1-96 | head -1
+
+You need to use the following command to create your ``root_password_sha2``::
+
+    echo -n yourpassword | sha256sum
+
+To be able to connect to Graylog you should set ``rest_listen_uri`` and ``web_listen_uri`` to the public host name or a public IP address of the machine you can connect to. More information about these settings can be found in :ref:`Configuring the web interface <configuring_webif>`.
+
+.. note:: If you're operating a single-node setup and would like to use HTTPS for the Graylog web interface and the Graylog REST API, it's possible to use :ref:`NGINX or Apache as a reverse proxy <configuring_webif_nginx>`.
+
+The last step is to enable Graylog during the operating system's startup::
+
+  $ sudo chkconfig graylog-server on
+  $ sudo systemctl daemon-reload
+  $ sudo systemctl start graylog-server.service
+
+The next step is to :ref:`ingest messages <ingest_data>` into your Graylog and extract the messages with :ref:`extractors <extractors>` or use :ref:`the Pipelines <pipelinestoc>` to work with the messages.
+
+
+
+Multiple Server Setup
```

Comment:
Why not "Cluster Setup" or "Multi-node Setup"?

Reply:
I just copied the centos guide and did not think about renaming parts - but we should rename this in all.

Reply:
✅